File: mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
     
     int h;
     do
     {
         printf("Height: ");  
         h=get_int();
       
     }while(h<0 || h>23);
     
     for(int i=2;i<=h+1;i++)
     {
         for(int j=1;j<=h-i+1;j++)
         {
             printf(" ");
         }
         for(int j=1;j<=i;j++)
         {
             printf("#");
         }
         printf("\n");
     }
}

File: pset6/mario.py
import cs50
def get_Height():
    while True:
        print("Height: ",end="")
        h = cs50.get_int()
        if h > 0 and h <= 23:
            return h   
     
h = get_Height();
for i in range(h):
    for j in range(0,h-i):
        print(" ",end="")
    for j in range(i+2):
        print("#",end="")
    print("")